velox/exec/tests/HashJoinTest.cpp

```diff
@@ -239,7 +327,23 @@ TEST_F(HashJoinTest, filter) {
       "((t_k0 % 100) + (u_k0 % 100)) % 40 < 20");
 }
 
-TEST_F(HashJoinTest, joinSidesDifferentSchema) {
+VELOX_INSTANTIATE_TEST_SUITE_P(
+    HashJoinTest,
+    ParallelJoin,
+    testing::ValuesIn(ParallelJoin::getTestParams()));
+
+// TODO: try to parallelize the following test case if possible.
+class NonParallelJoin : public HashJoinTest,
+                        public testing::WithParamInterface<TestParam> {
+ public:
+  NonParallelJoin() : HashJoinTest(GetParam()) {}
+
+  static std::vector<TestParam> getTestParams() {
+    return std::vector<TestParam>({TestParam{1}});
+  }
+};
+
+TEST_P(NonParallelJoin, joinSidesDifferentSchema) {
```

Comment:
Is it possible that before the followings are parallelized we just do
`TEST_F(HashJoinTest, ...)`
So that we don't need to create this additional class